test/algorithms/Jamfile

```diff
@@ -25,7 +25,6 @@ test-suite boost-geometry-algorithms
     [ run centroid_multi.cpp           : : : : algorithms_centroid_multi ]
     [ run comparable_distance.cpp      : : : : algorithms_comparable_distance ]
     [ run convert.cpp                  : : : : algorithms_convert ]
-    [ run convert_multi.cpp            : : : : algorithms_convert_multi ]
```

Comment:
Because it's not multi but also single to multi - it was more convenient to combine it into one source.